quickstart-semantic-search/README.md

```diff
@@ -1,3 +1,15 @@
+---
+page_type: sample
+languages:
+  - csharp
+name: C# semantic search with Azure AI Search
+description: |
+  This code sample provides the syntax for semantic search. It demonstrates an approach for adding a semantic configuration to a search index and query parameters to a query.
+products:
+  - azure
+  - azure-cognitive-search
+urlFragment: csharp-quickstart
```

Comment:
is this url-fragment unique??  Just checking

Reply:
Specify a urlFragment where possible ([see example](https://github.com/Azure-Samples/azure-search-postman-samples/tree/master/knowledge-store)). This ensures that the generated URL stub for the sample page on [learn.microsoft.com/samples](https://learn.microsoft.com/samples) is immutable. Otherwise, we will generate the link from the name of the sample, and if that changes, the link will change as well. No automatic redirection will be done in that case, which can result in broken links if others have linked to the sample from other locations.

Reply:
Doesn't say. I'll ask.

Reply:
@diberry  maybe just make it csharp-quickstart-semantic  -- since it's a URL, it has to be unique and the "csharp-quickstart" is alrady used in the v11 quickstart.